assets/admin/editor-wizard/steps/upgrade-step.js

```diff
@@ -1,25 +1,52 @@
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+
 /**
  * Upgrade step during course creation wizard.
  */
 const UpgradeStep = () => {
-	// TODO Implement this.
 	return (
-		<div>
-			<div>PENDING TO IMPLEMENT</div>
+		<div className="components-modal-columns">
+			<div className="components-modal-columns__description">
+				<p>
+					Do you want to sell this course? This requires Sensei Pro
```

Comment:
Just a reminder that we need to translate all texts using the i18n functions. 😉

Reply:
Fixed in [0d40106](https://github.com/Automattic/sensei/pull/5169/commits/0d40106a5438878a3c707e402aa5ef9a71ae811a). Thanks for the reminder!